### ISSUE TITLE
Hi, did you solve the stroke data problem now?

### ISSUE THREAD

**Issue by ghost** [StudioProcess/NeoSmartpenMac#3]
hi,
I happen to find your NeoSmartpenMac, so did you solve the problem now?
Can Your app get the stroke data from pen now?

https://github.com/NeoSmartpen/IOSSDK2.0/issues/5

Thanks.

**Comment by martinleopold** [comment]
Yes, it worked just as I described here: https://github.com/NeoSmartpen/IOSSDK2.0/issues/5#issuecomment-343923225

The app works just fine now, it runs as part of an exhibition installation for a few weeks now without any problems.

The PenTest app includes a WebSocket server that sends out pen events. Just point your browser to http://localhost:8080 for a little test app.
